src/spatialdata/_io/io_raster.py

```diff
@@ -67,7 +66,8 @@ def _read_multiscale(
     # and for instance in the xenium example
     encoded_ngff_transformations = multiscales[0]["coordinateTransformations"]
     transformations = _get_transformations_from_ngff_dict(encoded_ngff_transformations)
-    name = os.path.basename(node.metadata["name"])
+    # TODO: what to do with name? For now remove?
+    # name = os.path.basename(node.metadata["name"])
```

Comment:
I agree with @giovp in removing this, using a custom name has two downsides:
1) it makes the access to tensor inside the xarray object more verbose and non uniform, for instance one needs to write something like `sdata['my_image_name']['scale0']['my_image_name']`, instead of the standardized `sdata['my_image_name']['scale0']['image']`.
2) if the user assignes the image to a different name, like with `sdata['my_image2'] = sdata['my_image_name']`, the internal name will still renamin the old one, which is confusion.

@giovp for the sake of brevity I would actually call the name `im` instead of `image`; also in the case of labels I think `im` sounds better than `image`.